index.js

```diff
@@ -63,6 +66,12 @@ module.exports = (bookshelf, settings) => {
     initialize: function () {
       modelPrototype.initialize.call(this)
 
+      if (settings.sentinel) {
```

Comment:
That sounds reasonable, I'll look into it. For the time being, I was making use of the existing "settings" mechanism, which seems to be global -- the main example being `{ field: 'deletedAt' }` from the readme, which applies everywhere. Would you consider this a blocker for this commit, or could it be added in a separate one later? My impression is that if unique indexes are required on any table, they should be supported on every table.